### DISCUSSION TITLE
How do you read an rMQR code?

### DISCUSSION THREAD

**Discussion by hacktronics** [OUDON/rmqrcode-python#44] [General]
How do you read an rMQR code? Without any reader, what is the usage of such code.

**Comment by OUDON** [comment]
There is no reader for personal use yes. However [this device](https://www.denso-wave.com/en/adcd/product/scanner/qk30.html) can read an rMQR code. I hope reader apps  will support an rMQR code.

**Comment by OUDON** [comment]
Now we can read an rMQR code by [QRQR app](https://www.denso-wave.com/en/system/qr/product/reader.html).

**Comment by GCardona** [comment]
hey @OUDON nice tool you wrote there! I am finding however that big rmQR codes will not be read by QRQR. I do not know whether that is a problem of the QRQR app or the library itself... have you encountered any issues reading rmQR codes of say R15x139?
